Logout/__test__/handler.test.ts

```diff
@@ -20,15 +34,39 @@ describe("Logout", () => {
     jest.clearAllMocks();
   });
 
+  test.each([aValidL2User, aValidExchangeUser])(
+    `GIVEN a valid user decoded from JWT
+        WHEN audit log failed
+        THEN the response is 500`,
+    async user => {
+      const mockAuditLog = jest
+        .spyOn(auditLog, "storeAuditLog")
+        .mockReturnValueOnce(TE.left(("" as unknown) as RestError));
```

Comment:
I would suggest this edit if you don't mind
```suggestion
      const anError = "anError";
      const mockAuditLog = jest
        .spyOn(auditLog, "storeAuditLog")
        .mockReturnValueOnce(TE.left(new RestError(anError)));
```